diff --git a/node_modules/expo-web-browser/android/build.gradle b/node_modules/expo-web-browser/android/build.gradle
index a3810bd..50bc6f6 100644
--- a/node_modules/expo-web-browser/android/build.gradle
+++ b/node_modules/expo-web-browser/android/build.gradle
@@ -38,7 +38,7 @@ buildscript {
 
 // Creating sources with comments
 task androidSourcesJar(type: Jar) {
-  classifier = 'sources'
+  archiveClassifier.set('sources')
   from android.sourceSets.main.java.srcDirs
 }
 
